CONTRIBUTING.md

```diff
@@ -4,19 +4,13 @@ Thank you for considering contributing to Devopness!
 
 We appreciate the help!
 
-All community contributions are welcome! Being them:
-- 🐛 Opening [a bug report](https://github.com/devopness/devopness/issues/new/choose)
-- 📚 Improving [our documentation](https://github.com/devopness/devopness/tree/main/docs)
-- 🆕 Submitting [ideas and feature requests](https://github.com/devopness/devopness/discussions/categories/ideas-feature-requests)
-- ❓ Asking questions and helping other Devopness users with open [discussions](https://github.com/devopness/devopness/discussions)
-... every small contribution matters a lot to our team and for the Devopness community!
-
-## Not ready to start contributing?
-No problem!
-If you like Devopness, but just don't have time to contribute yet, that's fine.
-There are other easy ways to support Devopness and show your appreciation:
+All community contributions are helpful to our team and for the Devopness community!
+
+## Before getting started ...
+Show your appreciation for Devopness 💜! It means a lot to our team:
 - ⭐ Star this repository
-- 📣 Post about Devopness on social networks, Discord, LinkedIn, Slack, Twitter, ...
+- Follow [Devopness page on LinkedIn](https://www.linkedin.com/company/devopness/)
```

Comment:
```suggestion
- [Sign up](https://app.devopness.com/signup/) for a FREE Devopness account
- Follow [Devopness page on LinkedIn](https://www.linkedin.com/company/devopness/)
```

I believe it makes more sense to mention creating a Devopness account under the "Before getting started... Show your appreciation for Devopness" section.